paper.css

```diff
@@ -9,19 +9,22 @@ body { margin: 0 }
 }
 
 /** Paper sizes **/
-body.A3           .sheet { width: 297mm; height: 419mm }
-body.A3.landscape .sheet { width: 420mm; height: 296mm }
-body.A4           .sheet { width: 210mm; height: 296mm }
-body.A4.landscape .sheet { width: 297mm; height: 209mm }
-body.A5           .sheet { width: 148mm; height: 209mm }
-body.A5.landscape .sheet { width: 210mm; height: 147mm }
+body.A3           .sheet { width: 297mm; height: 419mm;margin-left: auto; margin-right: auto; }
```

Comment:
And I write at this place to make paper displayed at center default. If you think this feature is optional write it in another line is OK.